app/views/rating/applications/_application_data.html.slim

```diff
@@ -0,0 +1,142 @@
+- ad = application.application_data
```

Comment:
Depending on how easy it is to handle this large file, one could also extract a `_student` partial here. Would require string-interpolating the keys however...